versioned_docs/version-2/platform/integration/bigcommerce/embedded-checkout.md

```diff
@@ -0,0 +1,103 @@
+---
+id: embedded-checkout
+title: Embedded Checkout
+sidebar_label: Embedded Checkout
+description: How to use the BigCommerce Embedded Checkout
+---
+
+## About
+
+The [BigCommerce Embedded Checkout](https://developer.bigcommerce.com/api-docs/storefronts/embedded-checkout/embedded-checkout-tutorial) allows customer to load an iframe version of the checkout. The benefits to this is it supports all the features and payment methods BigCommerce supports out of the box.
+
+## How to use
+
+To use the BigCommerce Embedded checkout you simply need to add it into your checkout components.
+
+With our core app this will be:
+
+`client/src/pages/shop/Checkout/Checkout.js`
+
+### 1. Import the component
+
+```js
+import { EmbeddedCheckout } from './components';
+```
+
+:::note Check the component exists
+Out of the box you should have a `client/src/pages/shop/Checkout/components/EmbeddedCheckout.js` file. If not, please create a new app using `@deity/create-falcon-app` and grab it from there.
```

Comment:
I guess it needs a specific version (v2)?

Reply:
these are the v2 docs. I've not tested with v3

Reply:
but potentially yes

Reply:
I meant "example/demo-v2"

Reply:
Very true

Reply:
Fixed